src/test/regress/pg_regress_multi.pl

```diff
@@ -105,6 +105,14 @@ ()
     'mitmproxy' => \$useMitmproxy,
     'help' => sub { Usage() });
 
+my $fixopen = "$bindir/postgres.fixopen";
+my @pg_ctl_args = ();
+if (-e $fixopen)
+{
+	push(@pg_ctl_args, "-p");
+	push(@pg_ctl_args, $fixopen);
+}
+
```

Comment:
the changes here come from https://github.com/citusdata/citus/pull/4837 so that I could run the tests on my development machine.